### ISSUE TITLE
Expected a valid URI, not 'local' in Firefox when loading playlists, albums, tracks, or artists

### ISSUE THREAD

**Issue by partofthething** [jaedb/Iris#749]
**Describe the bug**
When I go to load playlists in Iris, the screen keeps loading forever and the webconsole gives this traceback in Firefox. Happens on albums, artists, and tracks as well. It works fine in Chromium on the same machine. 

**To Reproduce**
1. Install mopidy and iris and spotify and local playlists
2. Open web ui
3. navigate to Playlists

**Expected behavior**
Should show playlists, albums, tracks, or artists. 

**Console log**
```
traceback: "Traceback (most recent call last):\n  
File \"/usr/lib/python3/dist-packages/mopidy/internal/jsonrpc.py\", line 131, in _handle_single_request\n
    result = self._unwrap_result(result)\n  
File \"/usr/lib/python3/dist-packages/mopidy/internal/jsonrpc.py\", line 228, in _unwrap_result\n
    result = result.get()\n
File \"/usr/lib/python3/dist-packages/pykka/_threading.py\", line 45, in get\n
    _compat.reraise(*self._data['exc_info'])\n  
File \"/usr/lib/python3/dist-packages/pykka/_compat/__init__.py\", line 29, in reraise\n
    raise value\n  
File \"/usr/lib/python3/dist-packages/pykka/_actor.py\", line 193, in _actor_loop\n
    response = self._handle_receive(envelope.message)\n
File \"/usr/lib/python3/dist-packages/pykka/_actor.py\", line 299, in _handle_receive\n
    return callee(*message.args, **message.kwargs)\n
File \"/usr/lib/python3/dist-packages/mopidy/core/library.py\", line 92, in browse\n
    validation.check_uri(uri)\n  
File \"/usr/lib/python3/dist-packages/mopidy/internal/validation.py\", line 130, in check_uri\n
    raise exceptions.ValidationError(msg.format(arg=arg))\nmopidy.exceptions.ValidationError: Expected a valid URI, not 'local'\n"
```

**Screenshots**
![image](https://user-images.githubusercontent.com/5360219/124402752-11419980-dce7-11eb-9087-576d18c3e071.png)

**Environment details:**
Versions:
* Firefox 89.0.2 
* OS: Ubuntu 21.04
```
Mopidy                  3.1.1
Mopidy-Beets            4.0.1
Mopidy-Iris             3.58.0
Mopidy-Local            3.2.0
Mopidy-MPD              3.1.0
Mopidy-MPRIS            3.0.3
Mopidy-Spotify          4.1.1
```

**Additional context**
I tried this with Firefox in safe mode so I'm sure it's not my privacy plugins this time. 

Playing individual songs from spotify still works, interestingly.

**Comment by jaedb** [comment]
I'm struggling to replicate this. Do you get any useful mopidy logs during startup to indicate that `mopidy-local` may have failed to start?

**Comment by partofthething** [comment]
Not that I see. Since it works as expected in a Chrome and Chromium browser using the same server I suspect mopidy-local is up. But this error message does sound server-side, so it's quite a mystery. 

I do see a spotify-related traceback but at a different time. 

```
Jul 30 11:13:33 orion mopidy[2414242]: Traceback (most recent call last):
Jul 30 11:13:33 orion mopidy[2414242]:   File "/usr/lib/python3/dist-packages/mopidy/core/playlists.py", line 16, in _backend_error_handling
Jul 30 11:13:33 orion mopidy[2414242]:     yield
Jul 30 11:13:33 orion mopidy[2414242]:   File "/usr/lib/python3/dist-packages/mopidy/core/playlists.py", line 68, in as_list
Jul 30 11:13:33 orion mopidy[2414242]:     playlists = future.get()
Jul 30 11:13:33 orion mopidy[2414242]:   File "/usr/lib/python3/dist-packages/pykka/_threading.py", line 45, in get
Jul 30 11:13:33 orion mopidy[2414242]:     _compat.reraise(*self._data['exc_info'])
Jul 30 11:13:33 orion mopidy[2414242]:   File "/usr/lib/python3/dist-packages/pykka/_compat/__init__.py", line 29, in reraise
Jul 30 11:13:33 orion mopidy[2414242]:     raise value
Jul 30 11:13:33 orion mopidy[2414242]:   File "/usr/lib/python3/dist-packages/pykka/_actor.py", line 193, in _actor_loop
Jul 30 11:13:33 orion mopidy[2414242]:     response = self._handle_receive(envelope.message)
Jul 30 11:13:33 orion mopidy[2414242]:   File "/usr/lib/python3/dist-packages/pykka/_actor.py", line 299, in _handle_receive
Jul 30 11:13:33 orion mopidy[2414242]:     return callee(*message.args, **message.kwargs)
Jul 30 11:13:33 orion mopidy[2414242]:   File "/usr/lib/python3/dist-packages/mopidy_spotify/playlists.py", line 24, in as_list
Jul 30 11:13:33 orion mopidy[2414242]:     return list(self._get_flattened_playlist_refs())
Jul 30 11:13:33 orion mopidy[2414242]:   File "/usr/lib/python3/dist-packages/mopidy_spotify/translator.py", line 252, in to_playlist_refs
Jul 30 11:13:33 orion mopidy[2414242]:     for web_playlist in web_playlists:
Jul 30 11:13:33 orion mopidy[2414242]:   File "/usr/lib/python3/dist-packages/mopidy_spotify/web.py", line 438, in get_user_playlists
Jul 30 11:13:33 orion mopidy[2414242]:     for page in pages:
Jul 30 11:13:33 orion mopidy[2414242]:   File "/usr/lib/python3/dist-packages/mopidy_spotify/web.py", line 419, in get_all
Jul 30 11:13:33 orion mopidy[2414242]:     result = self.get_one(path, *args, **kwargs)
Jul 30 11:13:33 orion mopidy[2414242]:   File "/usr/lib/python3/dist-packages/mopidy_spotify/web.py", line 413, in get_one
Jul 30 11:13:33 orion mopidy[2414242]:     result = self.get(path, cache=self._cache, *args, **kwargs)
Jul 30 11:13:33 orion mopidy[2414242]:   File "/usr/lib/python3/dist-packages/mopidy_spotify/web.py", line 101, in get
Jul 30 11:13:33 orion mopidy[2414242]:     f"{result.get('error','Unknown')}"
Jul 30 11:13:33 orion mopidy[2414242]: AttributeError: 'NoneType' object has no attribute 'get'
```

**Comment by partofthething** [comment]
After more experimentation I clicked the "refresh" button in the GUI and they showed up! So then I went in and delete the local indexeddb entirely from the cache and restarted and everything works as expected. The crazy thing is that firefox in safe mode also didn't work, where I thought I had a fresh state, but apparently not. Anyway this seems to not be an issue after clearing cache. sorry for the trouble.
